### ISSUE TITLE
Pressing "Login" loses email/password already typed

### ISSUE THREAD

**Issue by frabcus** [vector-im/riot-android#945]
I typed in my email/username/password, and then pressed "Login", which I thought was a button at the bottom.

It then flipped like changing a tab, and gave me a new empty form with everything I had entered lost.

Either keeping what I entered, or making it clearer from the design that I need to choose login first.

**Comment by sabrinaj** [comment]
@frabcus This is the repository of the Android app. Please report your issues about the web client at https://github.com/vector-im/riot-web/issues. Thank you.

**Comment by frabcus** [comment]
@sabrinaj This one is actually an Android app issue! Apologies for the other two that weren't.

**Comment by sabrinaj** [comment]
I'm not sure I understand how you lost what you entered. Was it when switching from registration screen to login screen?

**Comment by frabcus** [comment]
@sabrinaj yes, that's correct.

I entered my email and password into the registration form, then pressed login thinking that was the button login. It flipped me to a different form and I had to retype my (long, but that's another story!) password.

**Comment by ylecollen** [comment]
frabcus  : to summarize you the login/register

-> you tap first on register / login to select the expected action : the selected action has a green background i.e register or login

-> you write the expected fields and tap a second time to start the registration / login.

The login and the registration are two separated forms : you can switch back from one to the other and retrieve the dedicated form fields.

Does it make sense ?

**Comment by frabcus** [comment]
@ylecollen yes, that is what it does. I found it confusing, and it gave me the burden of having to rekey my password during onboarding.
